main.js

```diff
@@ -1,57 +1,80 @@
-class Book{
-    constructor(author, title){
-      this.author =author
-      this.title =title
-    }
-  }
-  // ui class:handle the ui
-  class UI{
-    
-    static addBookToList(book){
-        const list = document.querySelector('#tablebody');
-        const row =document.createElement('tr');
-        row.innerHTML =`
+function Book(author, title) {
+  this.author = author;
+  this.title = title;
+}
+// ui class:handle the ui
+class UI {
+  static addBookToList(book) {
+    const list = document.querySelector('#tablebody');
+    const row = document.createElement('tr');
+    row.innerHTML = `
         <td>${book.author}</td>
         <td>${book.title}</td>
         <td><input type="button" value="remove" class="delete" ></td>
   
         `;
-        list.appendChild(row);
-    }
-    static deleteBook(el){
-      if(el.classList.contains('delete')){
-        el.parentElement.parentElement.remove();
-    }
+    list.appendChild(row);
   }
+
+  static deleteBook(el) {
+    if (el.classList.contains('delete')) {
+      el.parentElement.parentElement.remove();
+    }
   }
-  
-  // add book event
-  document.querySelector('#form-Book').addEventListener('submit',
-  (e) =>{
-    // pervent actual submit
-  e.preventDefault();
-  // get form values
-    const author = document.getElementById("authorname").value;
-    const title = document.getElementById("bookname").value;
+}
+
+// add book event
+document.querySelector('#form-Book').addEventListener('submit',
+  (e) => {
+    // prevent actual submit
+    e.preventDefault();
+    // get form values
+    const author = document.getElementById('authorname').value;
+    const title = document.getElementById('titlename').value;
     // instatiate book
     const book = new Book(author, title);
     // adding book to ui
-  UI.addBookToList(book);
-   
-  });
-  
-  // remove event
-  document.querySelector('#tablebody').addEventListener('click', (e) => {
-    UI.deleteBook(e.target);
+    UI.addBookToList(book);
```

Comment:
- [ ] when reloading the page the books are not loaded again from local storage, this is because you are overwriting the local storage with the last book every time therefore there is no list of books to recollect. 
- [ ] it would also be easier if you would use a book array to save the books as well, what you can do is append the book to the array and save the entire array to the local storage again.